README.md

```diff
@@ -84,55 +50,81 @@ Options:
-      --osm-tags <FILE.tsv>
-          Path to a TSV file that contains one or more of `wikidata`, `wikipedia` columns.
+See [the list of available mirrors](https://dumps.wikimedia.org/mirrors.html) for other options. Note that most of them do not include the enterprise dumps; check to see that the `other/enterprise_html/runs/` path includes subdirectories with files. The following two mirrors are known to include the enterprise html dumps as of August 2023:
+- (US) https://dumps.wikimedia.your.org
+- (Sweden) https://mirror.accum.se/mirror/wikimedia.org
 
-          This can be generated with the `get-tags` command or `osmconvert --csv-headline --csv 'wikidata wikipedia'`.
+For the wikiparser you'll want the ["NS0"](https://en.wikipedia.org/wiki/Wikipedia:Namespace) "ENTERPRISE-HTML" `.json.tar.gz` files.
 
-      --wikidata-qids <FILE>
-          Path to file that contains a Wikidata QID to extract on each line (e.g. `Q12345`)
+They are gzipped tar files containing a single file of newline-delimited JSON matching the [Wikimedia Enterprise API schema](https://enterprise.wikimedia.com/docs/data-dictionary/).
 
-      --wikipedia-urls <FILE>
-          Path to file that contains a Wikipedia article url to extract on each line (e.g. `https://lang.wikipedia.org/wiki/Article_Title`)
+The included [`download.sh`](./download.sh) script handles downloading the latest set of dumps in specific languages.
+It maintains a directory with the following layout:
+```
+<DUMP_DIR>/
+├── latest -> 20230701/
+├── 20230701/
+│  ├── dewiki-NS0-20230701-ENTERPRISE-HTML.json.tar.gz
+│  ├── enwiki-NS0-20230701-ENTERPRISE-HTML.json.tar.gz
+│  ├── eswiki-NS0-20230701-ENTERPRISE-HTML.json.tar.gz
+│  ...
+├── 20230620/
+│  ├── dewiki-NS0-20230620-ENTERPRISE-HTML.json.tar.gz
+│  ├── enwiki-NS0-20230620-ENTERPRISE-HTML.json.tar.gz
+│  ├── eswiki-NS0-20230620-ENTERPRISE-HTML.json.tar.gz
+│  ...
+...
 ```
 
-It takes as inputs:
-- A wikidata enterprise JSON dump, extracted and connected to `stdin`.
-- A directory to write the extracted articles to, as a CLI argument.
-- Any number of filters passed:
-  - A TSV file of wikidata qids and wikipedia urls, created by the `get-tags` command or `osmconvert`, passed as the CLI flag `--osm-tags`.
-  - A file of Wikidata QIDs to extract, one per line (e.g. `Q12345`), passed as the CLI flag `--wikidata-ids`.
-  - A file of Wikipedia article titles to extract, one per line (e.g. `https://$LANG.wikipedia.org/wiki/$ARTICLE_TITLE`), passed as a CLI flag `--wikipedia-urls`.
+## Usage with Maps Generator
+
+To use with the [maps generator](https://github.com/organicmaps/organicmaps/blob/master/tools/python/maps_generator/README.md), see the [`run.sh` script](run.sh) and its own help documentation.
+It handles extracting the tags, using multiple dumps, and re-running to convert titles to QIDs and extract them across languages.
 
-As an example of manual usage with the map generator:
+As an example of manual usage with the maps generator:
```

Comment:
Does it basically repeat/document what is done in the run.sh?

Reply:
Yes, without the parallel job handling and logging